src/main/java/tryCatchFinally/Demo.java

```diff
@@ -0,0 +1,29 @@
+package tryCatchFinally;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Demo {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter numerator:");
+        int numerator = scanner.nextInt();
+        System.out.println("Enter denominator:");
+        int denominator = scanner.nextInt();
+
+        try {
+            new Arithmetic();
```

Comment:
What's the point to create an instance of a class without storing it to any variable?
`Arithmetic.divide` is a `static` method, so no instance required for it.

But in general, it's a best practice to declare services in instance methods, so that it's easiesr to test and inject dependencies.

Reply:
Updated